bond/bond.py

```diff
@@ -1,119 +1,105 @@
-BOND_DEVICE_ACTION_TOGGLE_LIGHT = "ToggleLight"
-
-# Relating to Fireplace (FP)
-BOND_DEVICE_ACTION_SET_FLAME = "SetFlame"
-BOND_DEVICE_ACTION_INCREASE_FLAME = "IncreaseFlame"
-BOND_DEVICE_ACTION_DECREASE_FLAME = "DecreaseFlame"
-
+from bond.const import (Actions,DeviceTypes,Directions,Brightness)
 
 class Bond:
     def __init__(self, bondIp, bondToken):
         self.bondIp = bondIp
         self.bondToken = bondToken
 
     def stop(self, deviceId):
-        return self.doAction(deviceId, BOND_DEVICE_ACTION_STOP)
+        return self.doAction(deviceId, Actions.STOP)
 
     # Relating to Generic Device (GX)
     def turnOn(self, deviceId):
-        return self.doAction(deviceId, BOND_DEVICE_ACTION_TURN_ON)
+        return self.doAction(deviceId, Actions.TURN_ON)
 
     def turnOff(self, deviceId):
-        return self.doAction(deviceId, BOND_DEVICE_ACTION_TURN_OFF)
+        return self.doAction(deviceId, Actions.TURN_OFF)
 
     def togglePower(self, deviceId):
-        return self.doAction(deviceId, BOND_DEVICE_ACTION_TOGGLE_POWER)
+        return self.doAction(deviceId, Actions.TOGGLE_POWER)
 
     # Relating to Motorized Shades (MS), Dimmers and Garage Doors
     def open(self, deviceId):
-        return self.doAction(deviceId, BOND_DEVICE_ACTION_OPEN)
+        return self.doAction(deviceId, Actions.OPEN)
 
     def toggleOpen(self, deviceId):
-        return self.doAction(deviceId, BOND_DEVICE_ACTION_TOGGLE_OPEN)
+        return self.doAction(deviceId, Actions.TOGGLE_OPEN)
 
     def close(self, deviceId):
-        return self.doAction(deviceId, BOND_DEVICE_ACTION_CLOSE)
+        return self.doAction(deviceId, Actions.CLOSE)
 
     def hold(self, deviceId):
-        return self.doAction(deviceId, BOND_DEVICE_ACTION_HOLD)
+        return self.doAction(deviceId, Actions.HOLD)
 
     def pair(self, deviceId):
-        return self.doAction(deviceId, BOND_DEVICE_ACTION_PAIR)
+        return self.doAction(deviceId, Actions.PAIR)
 
     def preset(self, deviceId):
-        return self.doAction(deviceId, BOND_DEVICE_ACTION_PRESET)
+        return self.doAction(deviceId, Actions.PRESET)
 
     # Relating to Ceiling Fan (CF)
     def setSpeed(self, deviceId, speed=3):
         return self.doAction(deviceId,
-                             BOND_DEVICE_ACTION_SET_SPEED,
+                             Actions.SET_SPEED,
                              {"argument": speed})
 
     def increaseSpeed(self, deviceId, speed=1):
         return self.doAction(deviceId,
-                             BOND_DEVICE_ACTION_INCREASE_SPEED,
+                             Actions.INCREASE_SPEED,
                              {"argument": speed})
 
     def decreaseSpeed(self, deviceId, speed=1):
         return self.doAction(deviceId,
-                             BOND_DEVICE_ACTION_DECREASE_SPEED,
+                             Actions.DECREASE_SPEED,
                              {"argument": speed})
 
+    def setDirection(self, deviceId, direction=Directions.FORWARD):
```

Comment:
I really wish this was a separate pr. In the future please don't include extra changes as it adds additional time needed to merge and review prs.

Reply:
I can pull setDirection, toggleDirection, and setBrightness into a separate PR after this if you want?

Reply:
We can leave it in this pr :). Next time it would be good to split out for discussion.